api/v1/login.go

```diff
@@ -0,0 +1,55 @@
+package v1
+
+// LoginRequest is the request for the login request.
+type LoginRequest struct {
```

Comment:
Shall we add the capacity information into the request, including how many cpus, memory in bytes, disks metadata, disks spaces in bytes, root filesystem in bytes and gpu count?